CHANGES

```diff
@@ -1,7 +1,8 @@
 CHANGES in GRASS GIS 8.x
 
-https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80
-https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures82
+- https://github.com/OSGeo/grass/tags
```

Comment:
What is this file even for? Can we just remove it? It doesn't seem very useful at this point.

Reply:
I think it is (old?) packaging style. See https://producingoss.com/en/packaging.html

Citation:
_There should also be a CHANGES file (sometimes called NEWS), explaining what's new in this release._

With a similar comment, see: https://the-hitchhikers-guide-to-packaging.readthedocs.io/en/latest/creation.html

However, nowadays there seems to be no need for that any more as we have the ChangeLog file.

Reply:
Looks like it broke the compilation, so maybe it needs to be done in a separate PR.

Reply:
You mean deleting a file breaks the CI?

Reply:
Let's see if 21b22015b39726087f350d8bead949ad78ddc201 solves it.

Reply:
BTW, this is an example of why I think CI should run on all changes or in other words that we should require CI for every PR which we do and it protected us here.